### PR TITLE
Microverse Project - Mint Sign Up Form

### PR DESCRIPTION
**Pull request by happiguru**
This project was build to handle users registration on Mint.com website. It is not the original version, It is clone version for educational purpose only.

### CODE REVIEW

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @happiguru  and @umairarshadbutt    :handshake: 

### Good job so far :clap:  your project looks like the original :medal_sports: 

### there are some changes to get the project approved :point_down:
- You must fix the linter issues, the project can not be approved if the checks are not passed:
Screenshot from 2020-06-09 01-39-02
Let fix these issues.
- Please remove the comments from your code:
screenshot 
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### Once you are done with the changes, ask for another code review.

**Comment by happiguru** [comment]
Have made the correction to the file

**Comment by happiguru** [comment]
re-opening a pull request

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Congratulations team 🎉. Your project is approved 🙌

Consider making these optional changes before merging.

File: README.md
Comment (tirthajyoti-ghosh) [inline README.md]: - Add a working live demo link.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
[OPTIONAL] Decrease the overall font-size of the texts.

Onto the next project 🚀

Happy coding 👨‍💻

File: README.md
Comment (umairarshadbutt) [reply README.md]: Hi Tirthajyoti
I added the live demo link. Please review it

**Pull request merged**
